fbench/src/fbench/fbench.cpp

```diff
@@ -200,6 +200,28 @@ FBench::StopClients()
     printf("\nClients Joined.\n");
 }
 
+namespace {
+
+const char *
+approx(double latency, const ClientStatus & status) {
+    return (latency > (status._timetable.size() / status._timetableResolution - 1))
+           ? "ms (approx)"
+           : "ms";
+}
+
+std::string
+fmtPercentile(double percentile) {
+    char buf[32];
+    if (percentile <= 99.0) {
+        snprintf(buf, sizeof(buf), "%2d  ", int(percentile));
```

Comment:
I think this will add 2 extra spaces in the 'backwards' case...

Reply:
Yes, that is fine. I just wanted to avid decimals  in the old percentiles